getting_started_modules.md

```diff
@@ -0,0 +1,167 @@
+## Getting Started
+
+### Goal of This Guide
+
+> :warning: Before anything, if you haven't, read the section `Description - what & why` in [README.md](/README.md).
```

Comment:
Maybe say why the reader should read the `Description` section?

Reply:
Done!